comptests/ha/test_ha_shutdown_cluster.py

```diff
@@ -72,41 +72,38 @@ def setup_class(cls):
 
     def setup_method(self):
         """
-        Setup Method
+        This function will be invoked prior to each test case.
         """
         LOGGER.info("STARTED: Setup Operations")
         LOGGER.info("Check the overall status of the cluster.")
-        self.restored = True
         resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
         if not resp[0]:
             resp = self.ha_obj.restart_cluster(self.node_master_list[0])
             assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Cluster status is online.")
-        pod_list = self.node_master_list[0].get_all_pods(pod_prefix=common_const.HA_POD_NAME_PREFIX)
-        pod_name = pod_list[0]
-        output = self.node_master_list[0].execute_cmd(
-            cmd=common_cmd.KUBECTL_GET_POD_CONTAINERS.format(pod_list[0]),
-            read_lines=True)
-        container_list = output[0].split()
-        for container in container_list:
-            res = self.node_master_list[0].send_k8s_cmd(
-                operation="exec", pod=pod_name, namespace=common_const.NAMESPACE,
-                command_suffix=f"-c {container} -- {common_cmd.SERVICE_HA_STATUS}", decode=True)
-            assert_utils.assert_true(res, "HA services are not running")
+        LOGGER.info("Checking if all the ha services are up and running")
+        resp = HAK8SCompLib.check_ha_services(self.node_master_list[0])
+        assert_utils.assert_true(resp, "HA services are not running")
         LOGGER.info("Done: Setup operations.")
 
     def teardown_method(self):
         """
         This function will be invoked after each test function in the module.
         """
         LOGGER.info("STARTED: Teardown Operations.")
-        if self.restored:
-            LOGGER.info("Cleanup: Check cluster status and start it if not up.")
-            resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
-            if not resp[0]:
-                LOGGER.debug("Cluster status: %s", resp)
-                resp = self.ha_obj.restart_cluster(self.node_master_list[0])
-                assert_utils.assert_true(resp[0], resp[1])
+        if self.restore_pod:
```

Comment:
This condition is not needed as this suite is about only cluster shutdown/start.

Reply:
But in test_scale_pod_verify_ha_alerts TC, if anything goes wrong after the scale down, pod has to be restored right?
Also I've made it false after scaling up step.

Reply:
test_scale_pod_verify_ha_alerts doesn't seem to be related to F-8B.
If current suite is of F-8B only, then test needs to be moved to F-24ABC/E/F suite.
Please check.

Reply:
After receiving the cluster stop key, HA shouldn't send any alert to health monitor and should show skipping alert in monitor logs. In order to verify that we are performing pod scale down to verify the same.